axes/admin.py

```diff
@@ -38,11 +49,16 @@ class AccessAttemptAdmin(admin.ModelAdmin):
         "get_data",
         "post_data",
         "failures_since_start",
+        "expiration",
     ]
 
     def has_add_permission(self, request: HttpRequest) -> bool:
         return False
 
+    def expiration(self, obj: AccessAttempt):
+        if hasattr(obj, "expiration") and obj.expiration.expires_at:
```

Comment:
Maybe something like:

```suggestion
        return obj.expiration.expires_at if hasattr(obj, "expiration") else _("Not set")
```

Reply:
ya, in case of null=False for expires_at field, no more need to have a null check condition